.travis.yml

```diff
@@ -19,4 +19,5 @@ install:
 
 script:
     - ./run-checks --static || travis_terminate 1
+    - ./run-checks --unit || travis_terminate 1
```

Comment:
Sorry, this is the task https://github.com/snapcore/snapd/blob/master/tests/unit/go/task.yaml